messages/project.deploy.org.md

```diff
@@ -0,0 +1,197 @@
+# summary
+  
+Deploy source to an org.
+
+# description
+
+The source you deploy overwrites the corresponding metadata in your org. This command doesn’t attempt to merge your source with the versions in your org. If the command detects a conflict, it displays the conflicts but doesn’t complete the process. After reviewing the conflict, rerun the command with the --force-overwrite flag to overwrite the org.
+
+To run the command asynchronously, set --wait to 0, which immediately returns the job ID. This way, you can continue to use the CLI. By default the command waits to finish no matter how long the deployment takes.
```

Comment:
I might move the last sentence here to the beginning, it seems to frame up the basic expectation before talking about options/exceptions